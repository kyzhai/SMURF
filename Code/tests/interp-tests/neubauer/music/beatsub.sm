a = 1 $- 2
b = 2 $- 4.
c = 1. $- 2.

main = [[(1,0)$a],[(1,1)$b],[(1,2)$c]]

