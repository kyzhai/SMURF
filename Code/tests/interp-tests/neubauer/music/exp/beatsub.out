
Timing Resolution set to 4 PPQ

Instrument set to 48 on channel 0

Track 0:
 tick 0, channel 1: program change 48
 tick 0, channel 1: note C#2 on velocity: 90
 tick 8, channel 1: note C#2 on velocity: 0
 tick 8, channel 1: note C#3 on velocity: 90
 tick 10, channel 1: note C#3 on velocity: 0
 tick 10, channel 1: note C#4 on velocity: 90
 tick 22, channel 1: note C#4 on velocity: 0
 tick 22, end of track
PASSED SEMANTIC CHECKS
java -jar ./Lib/CSV2MIDI.jar a.csv a.midi
===== Program Successfully Finished =====
===== Result Writen to a.midi =====
